fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -315,10 +291,37 @@ case "$DistortionCorrection" in
 		fi
 		;;
 
-	${GENERAL_ELECTRIC_METHOD_OPT})
+	${GE_HEALTHCARE_LEGACY_METHOD_OPT})
 		if [ -z ${GEB0InputName} ]; then
-			log_Err_Abort "--fmapgeneralelectric must be specified with --dcmethod=${DistortionCorrection}"
+			log_Err_Abort "--fmapcombined must be specified with --dcmethod=${DistortionCorrection}"
+		fi
+    if [ -z ${deltaTE} ]; then
+      log_Err_Abort "--echodiff must be specified with --dcmethod=${DistortionCorrection}"
+    fi
+    # Check that FSL is at least the minimum required FSL version, abort if needed (and log FSL-version)
+    # This FSL version check is duplicated in global/scripts/FieldMapPreprocessingAll.sh
+    # The intention is to catch the error as early as possible. 
+    # GEHEALTHCARE_MINIMUM_FSL_VERSION defined in global/scripts/fsl_version.shlib
+    fsl_minimum_required_version_check "$GEHEALTHCARE_MINIMUM_FSL_VERSION" \
+      "For ${DistortionCorrection} method the minimum required FSL version is ${GEHEALTHCARE_MINIMUM_FSL_VERSION}. " 
+		;;
+  
+  ${GE_HEALTHCARE_METHOD_OPT})
+		if [ -z ${MagnitudeInputName} ]; then
+			log_Err_Abort "--fmapmag must be specified with --dcmethod=${DistortionCorrection}"
+		fi
+		if [ -z ${PhaseInputName} ]; then
+			log_Err_Abort "--fmapphase must be specified with --dcmethod=${DistortionCorrection}"
+		fi
+		if [ -z ${deltaTE} ]; then
+			log_Err_Abort "--echodiff must be specified with --dcmethod=${DistortionCorrection}"
 		fi
+    # Check that FSL is at least the minimum required FSL version, abort if needed (and log FSL-version)
+    # This FSL version check is duplicated in global/scripts/FieldMapPreprocessingAll.sh
+    # The intention is to catch the error as early as possible. 
+    # GEHEALTHCARE_MINIMUM_FSL_VERSION defined in global/scripts/fsl_version.shlib
+    fsl_minimum_required_version_check "$GEHEALTHCARE_MINIMUM_FSL_VERSION" \
+      "For ${DistortionCorrection} method the minimum required FSL version is ${GEHEALTHCARE_MINIMUM_FSL_VERSION}. "
```

Comment:
This has a mix of tabs and 2 spaces indent.  Sorry for not noticing earlier.  Since most of the surrounding stuff is tabs (and because I don't like 2 spaces indent, not distinct enough for me), I think I'd unify on tabs in this case (probably also fix the Philips section, too).